src/main/java/org/zalando/pazuzu/feature/FeatureRepository.java

```diff
@@ -4,7 +4,7 @@
 
 import java.util.List;
 
-public interface FeatureRepository extends CrudRepository<Feature, Integer> {
+public interface FeatureRepository extends CrudRepository<Feature, Integer>, FeatureRepositoryCustom {
```

Comment:
It is the recommended way of adding custom methods to repository according to documentation
http://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.single-repository-behaviour